doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst

```diff
@@ -81,4 +81,4 @@ Example output:
 
    tbbmalloc_proxy cannot replace memory allocation routines
    Success: free (ucrtbase.dll), byte pattern: <C7442410000000008B4424>
-   Fail: _msize (ucrtbase.dll), byte pattern: <E90B000000CCCCCCCCCCCC>
\ No newline at end of file
+   Fail: _msize (ucrtbase.dll), byte pattern: <E90B000000CCCCCCCCCCCC>
```

Comment:
What's changed here? Is it an accidental line break deletion?

Reply:
Nothing was changed there, not sure why it is displayed that way.